v3/e3x/compression.md

```diff
@@ -0,0 +1,61 @@
+# Channel Payload Compression
+
+Since channel packets are the most frequent and have a set of fixed well-known key/values in their JSON headers, both endpoints may support optional channel compression encoding to minimize the resources required.
+
+This is important in embedded/device networks where the MTU is small (BLE and 6lowpan), and may improve performance in other edge cases with frequent small packets.
+
+## `z` Handshake Signalling
+
+To indicate support of a channel payload compression any endpoint may include a `z` key with an unsigned integer value in the handshake.  The value `0` is the default and signals no support.
+
+The `z` value indicates how to decode/interpret the channel payload bytes immediately after decryption.  After any alternative processing the resulting value must still always be identical to a LOB packet with a JSON header and binary BODY, it is only to minimize encoding and not for use to include additional data in a payload.
+
+Both endpoints must include identical `z` in a confirmed handshake in order for it to be enabled on any channel packets using the resulting keys, and only that type of channel payload is supported when enabled.
+
+## `0` LOB encoded (default)
+
+All channel payload bytes are [LOB encoded](../lob).
+
+## `1` CBOR encoded
+
+The value `1` signals support of [CBOR based](http://cbor.io) payloads, the bytes are interpreted as a stream of CBOR values instead of LOB encoding.
+
+* first value is always channel id ("c", unsigned int)
```

Comment:
Ah I see, reread that section.  I feel with the example having low order numbers it might need to be more clear that this is not just the number always, but is correctly encoded cbor.  Maybe just have a higher number example?